monitoring/README.md

```diff
@@ -92,47 +157,113 @@ c) Upload the dashboard JSON files from your local `./grafana` sub-directory int
 a) API Server Monitoring: input into the following text boxes.
 - `NS`: name of the KubeStellar space namespace, e.g., *wds1-system*, *wds2-system*, *its1-system*, etc.
 - `APIService`: name of the service for a KubeStellar space APIserver, e.g., *wds1*, *wds2*, *vcluster*, etc.
+- `Instance`: hostname and port number for the target KubeStellar space APIserver, e.g., *10.244.0.18:9444*
 
     <img src="images/apiserver-monitoring.png" width="60%" height="80%" title="apiserver-monitoring"> 
 
-To monitor a WDS space (e.g., wds1) use the following parameters: `NS: wds1-system` and `APIService: wds1`. Likewise, to monitor a ITS space (e.g., its1): `NS: its1-system` and `APIService: vcluster`
+To monitor a WDS space (e.g., wds1) use the following combination of `NS/APIService` parameters: `NS: wds1-system` and `APIService: wds1`. Likewise, to monitor a ITS space (e.g., its1): `NS: its1-system` and `APIService: vcluster`
+
+**Note**: 
+   - You can also use the following combination of `NS/APIService` values to monitor the APIserver of the hosting cluster or a WEC cluster: `NS: default` and `APIService: apiserver`
+   - Use the following commands to obtain the Ip address of the APIServer in the hosting cluster, WDS (e.g., `wds1`) and ITS (e.g., `its1-system`) spaces:
+     ```bash
+     kubectl --context <hosting-cluster-context> get Endpoints -A | egrep "NAME| wds| vcluster |kubernetes"
+     ```
+     For example: 
+     ```bash
+     $ kubectl --context kind-kubeflex get Endpoints -A | egrep "NAME| wds| vcluster |kubernetes"
+     ```
+     Output:
+     ```bash
+     NAMESPACE         NAME                                                              ENDPOINTS                                                AGE
+     default           kubernetes                                                        172.18.0.2:6443                                          3d20h
+     its1-system       vcluster                                                          10.244.0.12:8443,10.244.0.12:8443                        3d20h
+     wds1-system       wds1                                                              10.244.0.21:9444                                         3d20h
+     ```
+     
+     If you're using kind clusters then use the following command to obtain the IP address of the host container for your cluster: `$ docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' <docker-container-name>`
+
+     For example: 
+     ```bash
+     docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' cluster1-control-plane
```

Comment:
Endpoints make this job easier too.

```console
mspreitz@mjs13 monitoring % kubectl --context cluster1 get Endpoints -n default 
NAME         ENDPOINTS         AGE
kubernetes   172.19.0.3:6443   25m
```

I see a different port number here (6443) than below (9444). In my testing just now, 6443 gets me data and 9444 does not.

Yes, it is the same IP address as from the `docker inspect`:

```console
mspreitz@mjs13 monitoring % docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' cluster1-control-plane
172.19.0.3
```

Reply:
Agreed! I updated to use endpoints instead of docker specific commands. Yes, port `# 6443`  is the correct one - the list of correct ports are also listed in the README instructions. 

`9444` is probably the port for your docker container itself.